### DISCUSSION TITLE
Per-field packed mode?

### DISCUSSION THREAD

**Discussion by k-ye** [taichi-dev/taichi#4160] [Ideas]
// We've probably discussed this before.

What's your thoughts on making the packed mode a per-ti.field configuration, rather than a global one? I can see a few benefits, if large and small fields are used simultaneously. @strongoier @turbo0628

**Comment by strongoier** [comment]
I'm also thinking about having a more fine-grained configuration. However, according to the design of the SNode system, packed mode can only be applied in a per-SNodeTree way.

**Reply by k-ye** [reply]
Oh yeah I forgot about that
